src/connectors/geo-search/connectGeoSearch.js

```diff
@@ -186,7 +192,7 @@ http://community.algolia.com/instantsearch.js/migration-guide
     }
 
     if (typeof widgetParams.precision !== 'undefined') {
-      warn(`
+      warning(`
```

Comment:
What happens if the condition isn't given (i.e. a truthy value / the error message)? There's more like this

Reply:
You need to pass `false` to the first argument to always warn. I forgot to change these ones.